scalableminds/webknossos
# Reduxify Skeletontracing

Mailable description of changes (needs to be understandable by webknossos mailing list people):
- replaced skeletontracing data structure implementation in preparation for dataset centric view

Steps to test:
- Start Skeleton Tracing, click on everything
- Run Unit Tests
- make sure the following modes still work:
  - volume tracings
  - public view mode / spotlight mode
  - readonly tracings
  - explorational tracings
  - task tracings

Issues:
- fixes #1709 

------
